SPECS/expat/expat.spec

```diff
@@ -1,7 +1,7 @@
 %define         underscore_version %(echo %{version} | cut -d. -f1-3 --output-delimiter="_")
 Summary:        An XML parser library
 Name:           expat
-Version:        2.5.0
+Version:        2.6.2
```

Comment:
Since this is a bigger version jump, could you verify all of the packages, which have a `BuildRequires` on one of the packages produced from this spec still build? It looks like the automated PR check has a functionality gap for toolchain packages like `expat` where it won't automatically try to rebuild dependents of toolchain packages, so for now I need to ask you to run a buddy build to verify this (or a local build). Sorry for the trouble!

Reply:
There are 27 different packages that have a `BuildRequires` on the packages produced by this spec. Would you suggest to build all 27 of those or is there another less complicated route we can take here?

Reply:
The expat PR should be good to merge in now - I ran a test build that built all dependent packages on exapt and they all built successfully (https://dev.azure.com/mariner-org/mariner/_build/results?buildId=535431&view=results). There were some failures but I reran those package builds independently and looks like they are failing for reasons other than the expat upgrade (pearl-XML-parser fails to build and avahi is blocked on fasttrack/2.0).